Makefile

```diff
@@ -180,7 +180,6 @@ bin/pulumi-java-gen: .pulumi-java-gen.version
 # - Run make ci-mgmt to apply the change locally.
 #
 ci-mgmt: .ci-mgmt.yaml
-	find .github/workflows/*.yml -type f ! -name "$(PACK)*.yml" -delete
```

Comment:
I'm surprised that this changes with the checkUpstreamUpgrade flag change

Reply:
This is an unrelated change from bumping ci-mgmt to latest.

Reply:
It was a recent change in ci-mgmt that had not done the nightly percolation. It's a legitimate change; I verified.